src/java.base/share/classes/java/lang/invoke/InnerClassLambdaMetafactory.java

```diff
@@ -344,10 +343,10 @@ private Class<?> generateInnerClass() throws LambdaConversionException {
             public void accept(ClassBuilder clb) {
                 clb.withFlags(ACC_SUPER + ACC_FINAL + ACC_SYNTHETIC);
                 clb.withInterfaceSymbols(interfaces);
+                clb.withVersion(CLASSFILE_VERSION, 0);
```

Comment:
Can't this be skipped as classfile api generates at the latest class file version by default?

Reply:
Fixed, thanks.